app/controllers/webchat.js

```diff
@@ -198,82 +214,183 @@ export default Ember.Controller.extend({
 				'customField3': this.get('field3value')
 			};
 
+
 			let storage = this.get('storageService');
 			if(storage.localStorageGet('relate.ui.useEmailAndPhoneForRWPLookupInWebChat')) {
 				chatData.email = this.get('email');
 			}
 
-			const customAttributes = this.get('customAttributes');
-			for (let attribute of customAttributes) {
-				if (attribute.name != '') {
-					chatData[attribute.name] = attribute.value;
+			// Add custom fields if V1
+			if (this.get('isV1') === true) {
```

Comment:
`if (this.get('isV1')`. We know its boolean.

Reply:
Thanks, I think that was leftover after I refactored from doing something else.